### PR TITLE
[FIX] Check for iterable before object.

### PR DESCRIPTION
**Pull request by garret-gunter**
Please prefix your pull request with one of the following: [FIX] [FEATURE].

### Changes proposed in this pull request:
- Fix for #313

### CODE REVIEW

File: src/Testing/FactoryBuilder.php [hunk 1]
```diff
@@ -213,18 +213,18 @@ protected function callClosureAttributes(array $attributes)
         return array_map(function ($attribute) use ($attributes) {
             if ($attribute instanceof \Closure) {
                 $entity = $attribute($attributes);
-                if (is_object($entity)) {
+	            if (is_iterable($entity)) {
```
Comment (patrickbrouwers) [inline src/Testing/FactoryBuilder.php]: This is a php7.1 method, but the current branch also still allows 7.0

Perhaps we can do is_array || isntanceof Traversable , as that's basically what is_iterable does ?

**Review by patrickbrouwers — CHANGES_REQUESTED** [review]

File: src/Testing/FactoryBuilder.php (hunk 1 above)
Comment (garret-gunter) [reply src/Testing/FactoryBuilder.php]: Fixed.  I always forget iterable was introduced in 7.1 and not 7.0.

**Comment by patrickbrouwers** [comment]
Can you fix CS please.

**Comment by garret-gunter** [comment]
Fixed cs.  Sorry for the delay.

**Review by patrickbrouwers — APPROVED** [review]

**Pull request merged**